src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -125,10 +125,6 @@ public void validate(ClientState state)
     {
         super.validate(state);
 
-        // since there are multiple guardrails guarding this, we first check that guardrails are globally enabled as an
-        // optimization for the case where they are disabled, so we don't have to do the same check on every guardrail
-        if (Guardrails.enabled(state))
-        {
             // Guardrail on table properties
```

Comment:
Fix indentation on this block now that you've removed the Guardrails.enabled check